keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -1121,11 +1188,14 @@ well as the [existing list] of feature gates.
 -->
 
 - [x] Feature gate (also fill in values in `kep.yaml`)
-  - Feature gate name: JobBackoffPolicy
+  - Feature gate name: PodDisruptionConditions
```

Comment:
This name might be confusing with PDBs, imo.

Reply:
What we define as a disruption is very close to the existing definition. With `TargetedForDisruption` we just standardize the way pods manifest disruption by the end state so that it is easy to setup the pod failure policy rules.

Reply:
PDBs are GA, so I don't think this name is concerning.

Reply:
Add the second feature gate

Reply:
Both gates are present listed in readme, adjusted indentention